litmus-portal/backend/graphql-server/pkg/cluster/util.go

```diff
@@ -21,3 +30,38 @@ func VerifyCluster(identity model.ClusterIdentity) (*model.Cluster, error) {
 	}
 	return &cluster[0], nil
 }
+
+func getIP() {
+
+    var kubeconfig *string
+    nodeIP := []corev1.NodeAddress{}
+
+    if home := homedir.HomeDir(); home != "" {
+        kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
```

Comment:
We need both in-cluster config and out-cluster.

Reply:
Thank you @gdsoumya  for your review, Yes i am working on that.

Reply:
@gdsoumya I improved for both in-cluster config and out-cluster.  Anything to improve?

Reply:
I don't think it is still using incluster-config, take reference from agent code by @rajdas98